Foundation/JSONSerialization.swift

```diff
@@ -25,6 +25,8 @@ extension JSONSerialization {
         
         public static let prettyPrinted = WritingOptions(rawValue: 1 << 0)
         public static let sortedKeys = WritingOptions(rawValue: 1 << 1)
+        public static let fragmentsAllowed = WritingOptions(rawValue: 1 << 2)
+        public static let withoutEscapingSlashes = WritingOptions(rawValue: 1 << 3)
```

Comment:
can we add this in a 5.2 or not since it's public and adding API?

Could we still do this if we make sure no new public types are added?

WDYT @millenomi ?

Reply:
You can always add any API that Darwin already has.

Reply:
(As is the case here.)

Reply:
That's great to hear, thank you :-)